src/hb-ot-layout-common.hh

```diff
@@ -2346,6 +2383,18 @@ struct FeatureVariationRecord
 {
   friend struct FeatureVariations;
 
+  void collect_lookups (const hb_set_t *feature_indexes,
+			hb_set_t       *lookup_indexes /* OUT */) const
+  {
+    return (this+substitutions).collect_lookups (feature_indexes, lookup_indexes);
```

Comment:
This feels like should be `base+substitutions` based on `sanitize ()` below.

Reply:
this is addressed in #2042.
Could you help review this PR #2043 again? I lost my 3rd commit in my previous rebase, now you should be able to review the 3rd part.

Reply:
Ok merged that also, rebase please, thanks